### ISSUE TITLE
Weird tooltip bolding with colons

### ISSUE THREAD

**Issue by Blakexx** [syncfusion/flutter-examples#94]
I was using a DateTime as the x axis and the tooltip that shows up for tracksballs is weirdly formatted when it is displaying a time. 

The DateTimeAxis class looks like this 

```
primaryXAxis: DateTimeAxis(
      dateFormat: DateFormat.jm()
 )
```

The TrackballBehavior class looks like this

```
trackballBehavior: TrackballBehavior(
      activationMode: ActivationMode.singleTap,
      enable: true,
      shouldAlwaysShow: true,
      tooltipSettings: InteractiveTooltip(
            color: Colors.white,
            format: "point.x : point.y"
      )
)
```

![image](https://cdn.discordapp.com/attachments/450497201958027274/692271590150242304/unknown.png)

**Comment by dharanidharandharmasivam** [comment]
Hi @Blakexx ,

Greetings from Syncfusion. The reported scenario has been fixed with our latest version (18.1.42). And we would like to let you know that, y value will bold in the trackball and this is our default behavior. To, resolve the issue kindly upgrade your chart widget package to the latest version from below.
https://pub.dev/packages/syncfusion_flutter_charts/versions/18.1.42 

Thanks,
Dharani.

**Comment by Blakexx** [comment]
Thanks!
